src/test/java/uk/gov/pay/products/persistence/dao/PaymentDaoIT.java

```diff
@@ -36,7 +36,28 @@ public void before(){
     }
 
     @Test
-    public void shouldSuccess_whenFindingAValidPayment() {
+    public void shouldFindAPaymentByGovukPaymentId() {
```

Comment:
Tempting to add a test where we don’t find the payment too.

Reply:
[Done](https://github.com/alphagov/pay-products/pull/1027/commits/a7e5cbf0cc85d728caa4fdee3aea86db232d47c0#diff-ba898b26a01716a28806e3548cd23a9f5ddd720281de8787dd483effba28becfR60)